201-load-balancer-ipv6-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "adminUsername": {
+            "value": "GEN-UNIQUE"
+        },
+        "adminPassword": {
+            "value": "GEN-UNIQUE"
+        },
+        "adminPassword": {
```

Comment:
adminPassword is in here twice - remove one and replace GEN-UNIQUE with GEN-PASSWORD (though I imagine either will work)